### PR TITLE
Update configure-advanced-delivery.md

### PR DESCRIPTION
**Pull request by ssaini-pm**
Added link to URL syntax format information

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/office-365-security/configure-advanced-delivery.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ The SecOps mailbox entries that you configured are displayed on the **SecOps mai
      - Single IP: For example, 192.168.1.1.
      - IP range: For example, 192.168.0.1-192.168.0.254.
      - CIDR IP: For example, 192.168.0.1/25.
-   - **Simulation URLs to allow**: Expand this setting and optionally enter specific URLs that are part of your phishing simulation campaign that should not be blocked or detonated by clicking in the box, entering a value, and then pressing Enter or selecting the value that's displayed below the box. You can add up to 10 entries.
+   - **Simulation URLs to allow**: Expand this setting and optionally enter specific URLs that are part of your phishing simulation campaign that should not be blocked or detonated by clicking in the box, entering a value, and then pressing Enter or selecting the value that's displayed below the box. You can add up to 10 entries. URL syntax follows format as described here [URL syntax for the Tenant Allow/Block List](https://docs.microsoft.com/en-us/microsoft-365/security/office-365-security/tenant-allow-block-list?view=o365-worldwide#url-syntax-for-the-tenant-allowblock-list).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/configure-advanced-delivery.md]:
```suggestion
   - **Simulation URLs to allow**: Expand this setting and optionally enter specific URLs that are part of your phishing simulation campaign that should not be blocked or detonated by clicking in the box, entering a value, and then pressing Enter or selecting the value that's displayed below the box. You can add up to 10 entries. For the URL syntax format, see [URL syntax for the Tenant Allow/Block List](/microsoft-365/security/office-365-security/tenant-allow-block-list?view=o365-worldwide#url-syntax-for-the-tenant-allowblock-list).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @ssaini-pm, kindly commit the suggested changes if you agree with them. Many thanks.

**Comment by yogkumgit** [comment]
@chrisda please merge this PR. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**